### PR TITLE
Mass update to roxygen2, NAMESPACE pkgdown site + hexsticker

### PR DESCRIPTION
**Pull request by fontikar**
Addressing #11 and #17 and #8
- Had to re-document exported data to meet best practices
- NAMESPACE is not fully generated sing roxygen tags
- Added a hex sticker in README and pkgdown website
- Package passing  '-as-cran' R CMD CHECK

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -10,3 +10,6 @@ test/
 ^\.github$
 ^LICENSE\.md$
 ignore/
+^doc$
+^Meta$
+ignore/
```
Comment (dfalster) [inline .Rbuildignore]: Missing new line

**Review by dfalster — CHANGES_REQUESTED** [review]
Hi @fontikar 

Wow - well done converting all these docs! Looks promising. Two question

1. Where was the function documentation previously stored? I can see lots of additions with roxygen, but no removals. did I miss something?
2. Tests are failing, can we fix before merging?

**Comment by fontikar** [comment]
Hey @dfalster 
> Wow - well done converting all these docs! Looks promising. Two question
> 
> 1. Where was the function documentation previously stored? I can see lots of additions with roxygen, but no removals. did I miss something?
I actually suspect Remko may have hand written those as .Rd files in man/ as there was never a doc/ dir until I came along!

> 2. Tests are failing, can we fix before merging?
Ahhh I just saw these! Yikes will work on this next!!!

**Comment by dwarton** [comment]
regarding 1., yes .Rd files were manually written before, so no removals because roxygen would have overwritten these

File: .Rbuildignore (hunk 1 above)
Comment (fontikar) [reply .Rbuildignore]: Arghhh, one day I will get this right!

Comment (fontikar) [reply .Rbuildignore]: I've realised what I've done wrong in the past. I normally just hit 'Enter' and save the file but in my recent commit ba0faba8 I've hit enter twice and save to force a new line! A late lesson learned!

**Comment by fontikar** [comment]
Hey @dfalster

> 2. Tests are failing, can we fix before merging?

I suspect many of the tests are failing because we need to regenerate the contents of man/ (all the .Rd files) which will fix a lot of the errors pertaining to plotting utils functions e.g. makeLogMinor, seqLog, defineAxis. These have been made internal as they are called only by plot.sma. 

I suggest merging and for me to fix and errors on master!

**Comment by fontikar** [comment]
Wahoo! merging now!

**Pull request merged**